HousingRegisterApi/V1/Domain/Address.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace HousingRegisterApi.V1.Domain
+{
+    public class Address
```

Comment:
Should the interface that backs these classes be tied somehow to the Typescript interfaces that we have in the frontend?

Reply:
They are defined in a swagger doc, which should provide guidance to match.